test/e2e/federated-ingress.go

```diff
@@ -104,7 +105,11 @@ var _ = framework.KubeDescribe("Federated ingresses [Feature:Federation]", func(
 			defer func() { // Cleanup
 				By(fmt.Sprintf("Deleting ingress %q in namespace %q", ingress.Name, ns))
 				err := f.FederationClientset_1_5.Ingresses(ns).Delete(ingress.Name, &v1.DeleteOptions{})
-				framework.ExpectNoError(err, "Error deleting ingress %q in namespace %q", ingress.Name, ns)
+				framework.ExpectNoError(err, "Error deleting ingress %q/%q in federation", ns, ingress.Name)
+				for clusterName, cluster := range clusters {
+					err := cluster.Ingresses(ns).Delete(ingress.Name, &v1.DeleteOptions{})
```

Comment:
Regarding your first comment, I followed the existing style use for the deletion of the federated ingresses.

Agreed about the GCE resources.  I'll add that in a separate PR.